cwltool/sandboxjs.py

```diff
@@ -1,12 +1,11 @@
-import cStringIO
```

Comment:
it looks impossible to make that type annotation work properly with Python3, the stringio stuff works quite different now.